tests/util/test_graph_hashing.py

```diff
@@ -37,14 +37,11 @@
 from __future__ import annotations
 
 import networkx as nx
-import pytest
 
+from pymatgen.util.graph_hashing import weisfeiler_lehman_graph_hash, weisfeiler_lehman_subgraph_hashes
 
-def test_graph_hash():
-    pytest.importorskip("emmet.core.graph_hashing")
```

Comment:
need confirm on this, [I'm not seeing `graph_hasing` from `emmet`](https://github.com/materialsproject/emmet/tree/main/emmet-core/emmet/core) but `pymatgen.util.graph_hashing`, perhaps these are relocated at some point?

https://github.com/materialsproject/pymatgen/blob/668f1aaffcaa0f3f56872b6eff93f27366acb356/src/pymatgen/util/graph_hashing.py#L76

https://github.com/materialsproject/pymatgen/blob/668f1aaffcaa0f3f56872b6eff93f27366acb356/src/pymatgen/util/graph_hashing.py#L151